requirements.txt

```diff
@@ -6,9 +6,9 @@ django-x509~=0.9.0
 django-taggit~=1.3.0
 django-loci>=0.3.1,<0.4.0
 django-flat-json-widget~=0.1
-openwisp-utils~=0.6.0
+openwisp-utils[rest]~=0.6.0
```

Comment:
@nemesisdesign I had to add this change otherwise pip was not installing django-filters from openwisp_utils[rest]. I tried removing openwisp_utils altogether, thinking that it already a dependency of openwisp_notification. But, that didn't work either.